cpp/tests/groupby/group_count_scan_test.cpp

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <tests/groupby/groupby_test_util.hpp>
+
+#include <cudf_test/base_fixture.hpp>
+#include <cudf_test/column_wrapper.hpp>
+#include <cudf_test/type_lists.hpp>
+
+#include <cudf/detail/aggregation/aggregation.hpp>
+
+namespace cudf {
+namespace test {
+template <typename V>
+struct groupby_count_scan_test : public cudf::test::BaseFixture {
+};
+
+TYPED_TEST_CASE(groupby_count_scan_test, cudf::test::AllTypes);
+
+// clang-format off
```

Comment:
I know turning clang-format off is fairly prolific in the other group-by gtests source files. But since these are new files ...
Would you consider just placing these markers around the specific lines where they are helpful  instead of the entire (or rather most) of the file?

Reply:
Done.